calico_node/startup/startup.go

```diff
@@ -888,13 +888,23 @@ func ensureGlobalBGPConfig(c *client.Client, key, def string) error {
 	if val, assigned, err := c.Config().GetBGPConfig(key, ""); err != nil {
 		return err
 	} else if !assigned {
-		return c.Config().SetBGPConfig(key, "", def)
+		return checkConflictError(c.Config().SetBGPConfig(key, "", def))
 	} else {
 		log.WithField(key, val).Debug("Global BGP value already assigned")
 		return nil
 	}
 }
 
+// checkConflictError checks to see if the given error is of the type ErrorResourceUpdateConflict
+// and ignore it if so. This is to allow our global configs to ignore conflict from multiple Nodes
+// trying to set the same value at the same time.
+func checkConflictError(err error) error {
+	if _, ok := err.(errors.ErrorResourceUpdateConflict); ok {
```

Comment:
@heschlie before we merge this can we at least add an info log here?

Something along the lines of "Ignoring conflict when attempting to set config %s=%s"